### PR TITLE
backend: add unit tests for eth address>account code matching

### PR DESCRIPTION
**Pull request by Tomasvrba**

### CODE REVIEW

File: backend/coins/eth/account_test.go [hunk 1]
```diff
@@ -158,3 +159,40 @@ func TestTxProposal(t *testing.T) {
...
+	acct := newAccount(t)
+	defer acct.Close()
+	acct.Synchronizer.WaitSynchronized()
+
+	// Test invalid Ethereum address
+	t.Run("Invalid Ethereum address", func(t *testing.T) {
+		matches, err := acct.MatchesAddress("invalid_address")
+		require.Error(t, err)
+		require.Equal(t, matches, false)
+		assert.Contains(t, err.Error(), "invalidAddress")
```
Comment (benma) [inline backend/coins/eth/account_test.go:173]: can check here that it is actually `errors.ErrInvalidAddress` 

    require.Equal(s.T(), errors.ErrInvalidAddress, errp.Cause(err))

**Review by benma — CHANGES_REQUESTED** [review]
thx!

**Comment by Tomasvrba** [comment]
Not sure why CI failed because of locize, there aren't any changes that should affect that here and running make locize-fix didn't change any files

**Comment by thisconnect** [comment]
I think we merged a formatting issue and probably due to nodejs not running properly. nodejs should be fixed on master and will be merged into staging-walletconnect

If I run locize fix I get changes, what output did you get, did locize even run?

instead of installing locize globally you can also do `npx locize-cli format frontends/web/src/locales --format json`

> Not sure why CI failed because of locize, there aren't any changes that should affect that here and running make locize-fix didn't change any files

**Comment by benma** [comment]
@Tomasvrba maybe a rebase will fix it? seems like it is fixed in the staging branch.

**Comment by Tomasvrba** [comment]
> @Tomasvrba maybe a rebase will fix it? seems like it is fixed in the staging branch.

should be all fixed up now

**Pull request merged**